pkg/agent/controller/networkpolicy/fqdn_test.go

```diff
@@ -744,3 +747,111 @@ func TestOnDNSResponse(t *testing.T) {
 		})
 	}
 }
+func TestFQDNCacheMinTTL(t *testing.T) {
+	currentTime := time.Now()
+	testIP := "192.168.1.1"
+	testFQDN := "fqdn-test-pod.lfx.test"
+	tests := []struct {
+		name            string
+		expectedTTL     time.Time
+		fqdnCacheMinTTL uint32
+		dnsMsg          *dns.Msg
+	}{
+		{
```

Comment:
we should test the IPv6 case (`dns.AAAA`) as well, just because it is a different code path